content/resources/faq.md

```diff
@@ -47,3 +48,11 @@ We don't think so and would be delighted to receive data from very large project
 We don't think this will be a problem because GitBOM does not include any metadata, such as provenance, timestamp, and licence -- the domain of [SBOM](/glossary/sbom)s.
 
 While duplicate hashes of empty files and regularly copied files (such as LICENSE files) are guaranteed to occur, this does not affect the security properties of GitBOM.
+
+## Why not use [Software Heritage Foundation](https://www.softwareheritage.org/) identifiers?
```

Comment:
The explanation below doesn't answer the question asked. IMO either the question should be changed or the answer should be more specific.

Reply:
I took your feedback below :)  Many thanks :)